core/src/Cuda/Kokkos_Cuda_Parallel_Team.hpp

```diff
@@ -47,6 +47,12 @@ extern bool show_warnings() noexcept;
 
 namespace Impl {
 
+// Cuda Teams use (team_size + 2)*sizeof(double) shared memory for team
+// reductions they also use one in64_t static shared memory for a shared ID
+// Furthermore, they use additional scratch memory in some reduction scenarios,
+// which depend on the size of the value_type and is NOT captured here.
+constexpr size_t cuda_team_max_reserved_shared_mem = (1024 + 2) * 8 + 8;
```

Comment:
Why not use the `sizeof` expressions shown in the comments?

Reply:
Yeah I guess I could do that.